### DISCUSSION TITLE
Show Restaurant POI earlier in car profile

### DISCUSSION THREAD

**Discussion by NotSoImportant** [osmandapp/OsmAnd#11582] [Maps Related]
### Description
https://www.openstreetmap.org/node/275324142
In car profiles shows up with min-zoom 18
In all other profiles with min-zoom 16

### Your Environment
OsmAnd Version: 3.9.10 
Android version: 7

**Maps used (online or offline):**   
Germany - Baden-Württemberg - Reutlingen

**Comment by sonora** [comment]
By design, I think, the car profile is optimized to show street/traffic related item in a de-cluttered fasion, not necessarily restaurants.

**Comment by NotSoImportant** [comment]
Yes that would of course sound legit, however there are other pois shown in zoom 18 for car profiles, that e.g. **I consider**

 less important for car navigation that are
- leasure=pitch
- sport=equestrian

it definitively makes sense for
- parking
- speed cam
- car repair
- car charger
- car wash

also helpful
- supermarket - I see restaurant in the same category of POIs

discussable
- fire brigade
- castle
- museum (interestingly castle wins for cars, but for others museum wins: https://osmand.net/go?lat=48.53280&lon=9.14896&z=16) 
- theater
- community center
- kindergarden
- shool

all these are currently shown in car profiles zoom 18 and more... So why not restaurant?

**Comment by NotSoImportant** [comment]
Let's put it like this: I don't want to discuss usefulness of all these, but I think restaurants are same important than supermarkets - (and they are; so please don't hide supermarkets as the fix, but instead also show restaurants ;-)

**Comment by vshcherb** [comment]
Small note: restaurants are very popular POI in contrast of Parks etc and mostly it is a point POI, so it can't be filtered by area for example. Obviously in city centers map would be cluttered with such POI's

**Comment by sonora** [comment]
Yes, exactly. Potentially very high densities can cause a lot more issues than less frequent ones like supermarkets or even areal ones like pitches or parks.

**Comment by NotSoImportant** [comment]
Ok I tried it in a bigger city and agree. Thanks.
